Базовая стилизация всех страниц

verrtiga opened the pull request:
---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2085107/tasks/6)

:boom: https://htmlacademy-htmlcss.github.io/2085107-sedona-34/4/

verrtiga committed:
Добавляет шрифты и классы

verrtiga committed:
Закончил стилизацию index

verrtiga committed:
Закончил стилизацию каталога

verrtiga committed:
добавляет sans-serif

verrtiga committed:
Добавляет фоны для преимуществ

verrtiga committed:
Поправил цвета

Zakamsky commented on styles/styles.css:
```diff
@@ -0,0 +1,233 @@
+    font-family:"PT Sans" ;
+    font-style: normal;
+    font-weight: 400;
+    src: url(../fonts/ptsans-400.woff2) format("woff2");
```
с точки зрения читаемости кода и его однообразного оформления стоит либо в обоих случаях использовать кавычки, либо в обоих не использовать.
и кавычки во всем файле должны быть одного типа.

Zakamsky commented on styles/styles.css:
```diff
@@ -0,0 +1,233 @@
+    font-family:"PT Sans" ;
+    font-style: normal;
+    font-weight: 700;
+    src: url(../fonts/ptsans-700.woff2) format("woff2");
+font-display: swap;
```
а тут что то с отступом...

Zakamsky commented on styles/styles.css:
```diff
@@ -0,0 +1,233 @@
+    src: url(../fonts/ptsans-700.woff2) format("woff2");
+font-display: swap;
+}
+body {
+font-family: "PT Sans" sans-serif;
```
тут я прямо уверен что тебе редактор подчеркнул это строку, будь внимателен, это грубая ошибка из за нее шрифт не работает.

Zakamsky commented on styles/styles.css:
```diff
@@ -0,0 +1,233 @@
+background-color:white;
+color:black;
+}
+.button {
+font-weight: 700;
+font-size: 16px;
+line-height: 20px;
+text-transform: uppercase;
+text-decoration: none;
+background-color: #756157;
+color: white;
+}
+.button-catalog {
```
и с отступами опять беда...

Zakamsky commented on styles/styles.css:
```diff
@@ -0,0 +1,233 @@
+background-repeat: no-repeat;
+background-size: 100% auto;
+width: 1200px;
+height: 485px;
```
Почти всегда лучше использовать `min-height` вместо `height` т.к. второе жестко фиксирует размер и не дает ему изменится даже если контента вдруг станет больше, т.е. при переполнении лишний текст просто выпадет из блока и все поедет...

Zakamsky commented on styles/styles.css:
```diff
@@ -0,0 +1,233 @@
+color: #000000;
+}
+.discription-sedona-litle {
+font-weight: 400;
```
Это ведь значение по умолчанию, как и общее значение шрифта - если сам тэг не имеет вендорных стилей, или мы ранее не перебили это значение, нет смысла его везде указывать.

Zakamsky reviewed (changes requested):
В целом все неплохо, но только  грубая ошибка в указании шрифта для `body` привела к тому что везде используется стандартный шрифт (я сейчас закину в чат видео как проверить какой шрифт используется).
Еще важный момент - не все тэги наследуют семейство шрифта, у некоторых есть вендорные свойства и стоит проверить что шрифт везде работает. наиболее капризные это кнопки и элементы форм.
Давай починим шрифт и можно двигаться дальше.

Пара уточнений что можно было бы сделать уже на этом этапе:
- сбросить отображения списка у тех списков где оно будет ненужно
- сразу оформить код правильно что бы потом твои же глаза не страдали.

verrtiga committed:
Исправил ошибки

Отформатировал код, исправил разнообразность кавычек, шрифт теперь работает, добавил шрифт к инпутам и батонам, исправил height на min-height

Zakamsky reviewed (approved):
Так намного лучше.
Апрув.
